terraform_modules/states_framework/backend.tf

```diff
@@ -0,0 +1,68 @@
+resource aws_s3_bucket tfstates {
+  bucket              = "nr-coreint-canaries-tfstates"
+  object_lock_enabled = true
+  force_destroy     = true # Allow to destroy all the versions automatically created.
+}
+
+resource aws_s3_bucket_versioning tfstates {
+  bucket = aws_s3_bucket.tfstates.bucket
+
+  versioning_configuration {
+    status = "Enabled"
+  }
+}
+
+resource "aws_s3_bucket_ownership_controls" tfstates {
```

Comment:
Add quotes or not but keep consistency, please.

Reply:
Fixed